tests/test_model_serializer.py

```diff
@@ -169,7 +170,7 @@ class Meta:
                 file_path_field = FilePathField(path='/tmp/')
         """)
 
-        self.assertEqual(unicode_repr(TestSerializer()), expected)
```

Comment:
Addressing any windows test failures should probably be treated as a separate PR.
I assume that's what this change is about, right?